playbooks/gce/openshift-cluster/config.yml

```diff
@@ -15,10 +15,10 @@
 
 - include: ../../common/openshift-cluster/config.yml
   vars:
-    g_etcd_group: "{{ 'tag_env-host-type-' ~ cluster_id ~ '-openshift-etcd' }}"
-    g_lb_group: "{{ 'tag_env-host-type-' ~ cluster_id ~ '-openshift-lb' }}"
-    g_masters_group: "{{ 'tag_env-host-type-' ~ cluster_id ~ '-openshift-master' }}"
-    g_nodes_group: "{{ 'tag_env-host-type-' ~ cluster_id ~ '-openshift-node' }}"
+    g_etcd_group: "{{ groups['tag_host-type-openshift-etcd'] | intersect(groups['tag_environment_' ~ cluster_id}}"
+    g_lb_group: "{{ 'tag_host-type-openshift-lb' | intersect(groups['tag_environment_' ~ cluster_id] }}"
+    g_masters_group: "{{ 'tag_host-type-openshift-master' | intersect(groups['tag_environment_' ~ cluster_id] }}"
+    g_nodes_group: "{{ 'tag_host-type-openshift-node' | intersect(groups['tag_environment_' ~ cluster_id] }}"
```

Comment:
I agree with @detiber, these vars should be renamed to g_*_hosts.

I really like the idea of having these be simple host lists as they don't seem to be being used as host groups in any playbooks (in this repo, anyway).